### DISCUSSION TITLE
支持xbox吗？

### DISCUSSION THREAD

**Discussion by durjgdjajqsbdbfj** [ywmoyue/biliuwp-lite#491] [General]
### 问题描述

哔哩哔哩官方uwp版本，早已停止维护，无法登陆，正常使用，我非常希望能在xbox上观看B站，您的软件支持吗？

### 描述解决方案

操作方式和官方的uwp版本一样，用手柄

### 附加信息

无

### 代码贡献

否

**Comment by ywmoyue** [comment]
不支持

**Reply by durjgdjajqsbdbfj** [reply]
OK
